.gitignore

```diff
@@ -17,7 +17,6 @@ stage/
 local/
 eventgen_wsgi.conf
 *.log
-dist
```

Comment:
I think this should not be deleted, right?

Reply:
It was a duplicate line in the file.